docs/Interesting-Taps-and-Forks.md

```diff
@@ -24,6 +24,8 @@ Your taps are Git repositories located at `$(brew --repository)/Library/Taps`.
 
 *   [lifepillar/appleii](https://github.com/lifepillar/homebrew-appleii): Formulae for vintage Apple emulation.
 
+*   [gromgit/fuse](https://github.com/gromgit/homebrew-fuse): macOS FUSE formulae that are no longer available in Homebrew core.
```

Comment:
```suggestion
*   [gromgit/fuse](https://github.com/gromgit/homebrew-fuse): macOS FUSE formulae that are no longer available in `homebrew/core`.
```

_Very_ minor style suggestion 😅. The document references the `homebrew/core` tap in this format above, so it might be better to do the same here.

Reply:
Good point, thanks much, @nandahkrishna !